src/NuGetGallery/Controllers/PackagesController.cs

```diff
@@ -1134,8 +1144,19 @@ public virtual async Task<ActionResult> ListPackages(PackageListSearchViewModel
             var isPreviewSearchEnabled = _abTestService.IsPreviewSearchEnabled(GetCurrentUser());
             var searchService = isPreviewSearchEnabled ? _searchServiceFactory.GetPreviewService() : _searchServiceFactory.GetService();
 
+            if(!isSupportedSortBy(searchAndListModel.SortBy))
+            {
+                searchAndListModel.SortBy = GalleryConstants.SearchSortNames.Relevance;
+            }
+
+            var isDefaultSortBy = string.Equals(searchAndListModel.SortBy, GalleryConstants.SearchSortNames.Relevance, StringComparison.OrdinalIgnoreCase);
+            var isDefaultPackageType = searchAndListModel.PackageType == null || !searchAndListModel.PackageType.Any();
```

Comment:
You can simplify this as:

```suggestion
            var isDefaultPackageType = string.IsNullOrEmpty(searchAndListModel.PackageType);
```